main.js

```diff
@@ -0,0 +1,53 @@
+console.log("Javascript Milestone")
+var seed = []
+var space = [" "]
+//var ast = document.getElementById("treeType").value
+var brake = ["\n"]
+var treeParams = {}
+var enter = document.getElementById("treeHeight").addEventListener("keydown", function(){
+	console.log("event", event);
+	if (event.key === "Enter"){
+		treeParams = {treeHeight : document.getElementById("treeHeight").value, treeType : document.getElementById("treeType").value};
```

Comment:
You could really clean up your code by setting your element variables on the global scope. Then you just have to refer to them when you want them.